### ISSUE TITLE
Hyperparameter inconsistency

### ISSUE THREAD

**Issue by orenavram** [berkindurmus/SLIViT#9]
It seems like the NoduleMNIST fine-tuning uses 20 heads:
_python slivit_train.py --dataset3d nodulemnist --task classification --bbpath /path/to/pretrained/backbone --nslc 28 --depth 5 --dim 64 --heads 20 --out\_dir /output/dir/to/save\_finetuned\_slivit/_ 

However, in the paper we reported that we use 10. Not sure if on purpose or mistake but anyway we can go with either as long as the code and text are consistent. LMK what do you prefer. Thanks
